src/components/WeaponComponent.vue

```diff
@@ -107,6 +107,8 @@ export default {
 
 		requirementTooltip(weapon, camouflage) {
 			const requirement = this.requirements[weapon.category][weapon.name][camouflage]
+			if (requirement && requirement.Challenge) return `${camouflage} - Level ${requirement.Level} - ${requirement.Challenge}`
```

Comment:
Is it possible to maybe lowercase the property names in the camouflage objects? So it's consistent with the other properties, for example in the weapons? I prefer to have properties lowercase, unless the property itself is needed to be shown in the UI.